manager/state/raft/raft.go

```diff
@@ -193,6 +189,7 @@ func NewNode(ctx context.Context, opts NewNodeOptions) *Node {
 		stopCh:              make(chan struct{}),
 		doneCh:              make(chan struct{}),
 		removeRaftCh:        make(chan struct{}),
+		stopped:             make(chan struct{}),
```

Comment:
It's quite different. To stop one could just call RemoveNode on leader and node will stop. I don't know if it's right behavior, but that's how it is :/